cpp/include/cudf/io/detail/parquet.hpp

```diff
@@ -105,48 +145,30 @@ class writer {
    * @brief Writes the dataset as per options provided.
    *
    * @param table Set of columns to output
-   * @param metadata Table metadata and column names
    * @param return_filemetadata If true, return the raw file metadata
    * @param column_chunks_file_path Column chunks file path to be set in the raw output metadata
-   * @param int96_timestamps If true, write timestamps as INT96 values
-   * @param stream CUDA stream used for device memory operations and kernel launches.
-   */
-  std::unique_ptr<std::vector<uint8_t>> write(
-    table_view const& table,
-    const table_metadata* metadata                = nullptr,
-    bool return_filemetadata                      = false,
-    const std::string column_chunks_file_path     = "",
-    std::vector<uint8_t> const& decimal_precision = {},
-    rmm::cuda_stream_view stream                  = rmm::cuda_stream_default);
-
-  /**
-   * @brief Begins the chunked/streamed write process.
-   *
-   * @param[in] pq_chunked_state Internal state maintained between chunks.
    */
-  void write_chunked_begin(struct pq_chunked_state& state);
+  std::unique_ptr<std::vector<uint8_t>> write(table_view const& table,
+                                              bool return_filemetadata,
+                                              const std::string column_chunks_file_path = "");
```

Comment:
This one somehow lost the reference.
```suggestion
                                              std::string const& column_chunks_file_path = "");
```